authentication-request-handler/pom.xml

```diff
@@ -0,0 +1,170 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.wso2.carbon.identity.framework.extended</groupId>
+	<artifactId>authentication.request.handler</artifactId>
+	<packaging>jar</packaging>
+	<version>1.0.0</version>
+	<name>sample-request-handler-extension</name>
+	<description></description>
+
+	<!-- TODO
```

Comment:
Shall we remove these if not needed?

Reply:
Fixed with c907fcc2a908e093c988f8bf86c54ce8e98b72c0